tests/testMixture.py

```diff
@@ -70,7 +70,6 @@ def testSwig(self):
         l1[2].weight = 0.5
         m1 = lsst.meas.modelfit.Mixture(1, l1)
         self.assertEqual(m1[0].weight, 0.5)
-        self.assertEqual(len(l1), 0)
```

Comment:
Right.  This used to consume the list's elements in Python, in what was almost certainly a premature optimization that tried to avoid copying.